database/transaction.go

```diff
@@ -12,19 +11,20 @@ import (
 )
 
 var (
-	tableConfigStoreName = "__genji.tables"
-	indexStoreName       = "__genji.indexes"
+	tableInfoStoreName = "__genji.tables"
+	indexStoreName     = "__genji.indexes"
 )
 
 // Transaction represents a database transaction. It provides methods for managing the
 // collection of tables and the transaction itself.
 // Transaction is either read-only or read/write. Read-only can be used to read tables
 // and read/write can be used to read, create, delete and modify tables.
 type Transaction struct {
-	db         *Database
-	Tx         engine.Transaction
-	writable   bool
-	tcfgStore  *tableConfigStore
+	db       *Database
+	Tx       engine.Transaction
+	writable bool
+
+	infoStore  *tableInfoStore
```

Comment:
We should be explicit here

```suggestion
	tableInfoStore  *tableInfoStore
```